challenges/leftJoin/left-join.js

```diff
@@ -0,0 +1,26 @@
+const leftJoin = (tableOne, tableTwo) => {
+  let res = [];
+  for(let i = 0; i < tableOne.bucket.length; i++) {
+    if(tableOne.bucket[i] !== undefined) {
+      res.push(tableOne.bucket[i]);
+    }
+  }
+  for(let i = 0; i < tableTwo.bucket.length; i++) {
+    if(tableTwo.bucket[i] !== undefined) {
+      res.forEach(kv => {
```

Comment:
I would try this with a for loop instead of a for each and see if that fixes the issue